server/server.js

```diff
@@ -1,41 +1,7 @@
-const express = require('express');
-const db = require('../database/index.js');
-const app = express();
+const app = require('./index.js');
 const port = 3001;
 
-app.use(express.static(__dirname + '/../client/public'))
```

Comment:
did you end up deleting all of this or does it show back up somewhere? it looks like info you would need eventually..

Reply:
No, the code's all there on my side